src/index.js

```diff
@@ -872,8 +872,10 @@ class ServerlessAppsyncPlugin {
     const flattenedMappingTemplates = config.mappingTemplates
       .reduce((accumulator, currentValue) => accumulator.concat(currentValue), []);
     return flattenedMappingTemplates.reduce((acc, tpl) => {
-      const reqTemplPath = path.join(config.mappingTemplatesLocation, tpl.request || `${tpl.type}.${tpl.field}.request.vtl`);
-      const respTemplPath = path.join(config.mappingTemplatesLocation, tpl.response || `${tpl.type}.${tpl.field}.response.vtl`);
+      const reqSuffix = tpl.kind === 'PIPELINE' ? 'before' : 'request';
+      const respSuffix = tpl.kind === 'PIPELINE' ? 'after' : 'response';
+      const reqTemplPath = path.join(config.mappingTemplatesLocation, tpl.request || `${tpl.type}.${tpl.field}.${reqSuffix}.vtl`);
+      const respTemplPath = path.join(config.mappingTemplatesLocation, tpl.response || `${tpl.type}.${tpl.field}.${respSuffix}.vtl`);
```

Comment:
@markvp This however **is** a breaking change.
People probably already rely on the fact that their `before`/`after` mapping templates are called `request`/`response`.

I see no problem in adding support for `.before`/`.after` suffixes but we must keep backward compatibility with `request`/`response`.

Reply:
Hmm. I understand the desire for backwards compatibility. Not sure the best way to implement that. Perhaps checking for file existence first?

Reply:
@markvp 🤔 Well, in CloudFormations (and in this plugin), even if in the doc they refer to it as `before` and `after`, the fields that hold these template paths are still called `request` and `response`.
So I think the right thing to do is to avoid complexity (and confusion) and stick to that.

Reply:
ok. removed before/after update for now